src/renderer/components/NodeSelectorPanel/NodeSelectorPanel.tsx

```diff
@@ -219,6 +219,10 @@ export const NodeSelector = memo(() => {
                                                 ? getSubcategories(categoryNodes)
                                                 : null;
 
+                                            if (!categoryNodes && !categoryIsMissingNodes) {
```

Comment:
Could you move this check above `const subcategoryMap = ...´? I think the check should be closer to the variables it checks.